package.json

```diff
@@ -79,6 +80,7 @@
     "yargs": "^17.5.1"
   },
   "resolutions": {
+    "@types/prettier": "2.6.0",
```

Comment:
2.6.1 and above requires typescript@^4.2.0. I don't know why this wasn't an issue before, but now `yarn test-snippets` was failing because of this.